Model/Data/OrderUpdate.php

```diff
@@ -85,6 +98,13 @@ public function update($order, $amwalOrderData, $historyComment = '', $sendAdmin
 
         try {
             $status = $amwalOrderData->getStatus();
+            if($trigger == 'PendingOrdersUpdate') {
+                $historyComment = __('Successfully completed Amwal payment with transaction ID %1 By Cron Job', $amwalOrderId);
+            } elseif($trigger == 'AmwalOrderDetails') {
+                $historyComment = __('Order status updated to (%1) by Amwal Payments webhook', $status);
+            } elseif($trigger == 'PayOrder') {
+                $historyComment = __('Successfully completed Amwal payment with transaction ID: %1', $amwalOrderId);
+            }
```

Comment:
While this might not be a case yet we could also add a generic `else` case to capture an unknown trigger. We can then add the trigger in the message to make debugging easier if needed